docs/tutorial/rust/src/ideas_for_the_reader.md

```diff
@@ -2,13 +2,13 @@
 
 The game is visually a little bare. Here are some ideas how you could make further changes to enhance it:
 
-* The tiles could have rounded corners, to look a little less sharp. The [border-radius](https://slint-ui.com/docs/rust/slint/docs/builtin_elements/index.html#rectangle)
-  property of *Rectangle* can be used to achieve that.
+-   The tiles could have rounded corners, to look a little less sharp. The [border-radius](https://slint-ui.com/docs/rust/slint/docs/builtin_elements/index.html#rectangle)
+    property of _Rectangle_ can be used to achieve that.
```

Comment:
Perhaps there's a way to configure prettier not to do this change and also make it less noisy? It seems to me that underscore is not recommended: https://www.markdownguide.org/basic-syntax/#italic-best-practices